### ISSUE TITLE
G> Create a clear picture of the coop and it's governance process(es)

### ISSUE THREAD

**Issue by pmoorman** [rchain/Members#379]
### **Context:**

@Ojimadu wrote:

> 
> There needs to be a clear picture of the structure of the coop. There seem to be quite some confusion especially when not coming from the board about who speaks for the coop, who approves what, what act can be said to be that of the coop and what act is not. As things stand now, most of the decision making is vested in the board of directors and the president.
> 
> At the 'RAM' meeting of 2018-02-07 @drbloom was of the [thought? opinion? fact?] that 'RAM' was somehow different from the core cooperative. That "RAM" was more of a bounty program put in place by greg and activities here on this repository may/can not be said explicitly to be that of the coop. He likened "RAM" to Reflective ventures and RChain holdings which work closely with the coop but not representative of the coop (That was how I understood the conversation between him and @wordsandstuff (brandon) on the T-shirt for the governance forum). @drbloom is a board member so I don't know if that was just his opinion or a sentiment shared by board, if the latter is true then this needs to be clearly pointed out.
> 
> Currently, there is a discussion on the #exec-committee channel on discord on mission statement, vision etc. -reminds me of the recent logo/rebranding talks- that got me wondering peradventure we arrive on a consensus for the mission and vision statements who get to approve them? The board? The committee? This issue #370 also falls into the same question above.

I think this is a widely felt pain, and something I've been thinking to work on, also as part of onboarding education for RAM / bounty members (see #253 for that)

### **Scope:**

Create a clear organisational breakdown of the RChain coop, and it's relationship to other parties. The resulting document should be usable in the context of onboarding & activating new members/collaborators, as well as serve as a guideline for existing members to navigate the 'political' realities of the structure we work in.

@dckc has suggested it should be reflected in one or more wiki entries, and also integrated into the onboarding flow, such as "welcome to the RAM" 

### **Problems it should solve:**

- Make it easy for understand what each term means  _::  "what is the RAM"_
- Show how things relate to each other  _::  "what is the difference between the coop and the RAM?"_
- Outline how decisions get made  _::  "how do we decide on a new logo?"_
- Help members find the right people/places to get their ideas & suggestions heard  _::  "I think we should redesign the website. With who should I talk about this?"_

### **Action items:**

- [ ] Draft "organisational diagram"
- [ ] Outline relationships between parties
- [ ] Lead discussion on these topics (either in comments here, or in e.g. #exec-committee)
- [ ] Define the political processes
- [ ] Create documents for e.g. the wiki
- [ ] Get everything signed off on...       ;-)
- [ ] Implement changes

**Comment by dckc** [comment]
I expect the energy around a membership working group to help address this.

Peter from Resonate drew a nice organizational diagram on a flip chart.

And Amanda (?) had a spreadsheet for people to sign up to get involved. I didn't volunteer there, but I did talk with @patrick727 and Jeremy about members.rchain.coop stuff; and Kevin Valentine asked if I'd help out, and I said I'm not sure how available I am but I would like to help.

**Comment by AyAyRon-P** [comment]
Do you want to refine the org chart we put together this past week @pmoorman and then post?

**Comment by pmoorman** [comment]
@dckc said:

> I expect the energy around a membership working group to help address this.

Can someone point me to further info about this?

Regardless, I think it would still really boost clarity (and therefore understanding) if we can make the way the coop functions more transparent. I believe transparency is a core property of any cooperative, so it would definitely fit there.

@AyAyRon-P and I have been doodling no a bit of a diagram to start with outlining just a bunch of terminology that's unclear to new(ish) members. I'll post it below, so everyone can already comment here.

THIS IS A DRAFT (!!!):

-------

- **RChain** 
references the core platform, core product, technology. This is the main brand. Analogy could be “Coca Cola” as the main brand, for instance. Or “Apple” as the main brand.

- **RChain Coop** 
is the steward of the platform. Think “The Coca Cola Company”, or “Apple Inc.”. Think of it as the shareholders / owners of the platform (although structured as members of a cooperative).

- **Rholang** 
is a programming language developed for RChain. This is similar to how “Swift” is the programming language for iOS for instance (in the Apple analogy).

- **Mercury** 
is a software release, and a version of the platform. In software terms, think of “OS X High Sierra” for Apple. In the physical world, an analogy could be “iPhone 7” for instance. After the iPhone 7 came the iPhone 8 (duh!), and after software release Mercury will come software release Venus.

- **RChain Holdings** 
is a for-profit that is in a joint-venture with the Coop. It creates value by bringing Enterprise users on the platform. Similar to “Enterprise Ethereum”, for instance.

- **RChain Ventures**  
is essentially a VC fund (funded through the coop) for providing funding to projects that could develop on the RChain platform.

- **Pyrofex** 
is a partner that does most of the software development for the platform. The “core dev team” works for Pyrofex. Pyrofex is a for-profit company.

- **The Board** 
is the highest entity within the Coop. This is similar to how every public company has a “Board of Directors”. The board is democratically elected by the Members of the Coop.

- **Committees** 
are another structure within the Coop, there are multiple committees. For instance: the “Executive Committee”. Each committee has a mandate, and a certain specific task to perform. Committees are appointed and (to the extend needed) democratically elected.

- **Greg Meredith** 
is the chairman of the Board of RChain Coop. Much like Steve Jobs was the CEO of Apple Inc., Greg is basically our “founder.”

- **Nash Foster** 
is the CEO of Pyrofex, and as such leads the core development team.

etc. etc.   (btw, it looks like an endless list here in Github). But I think a bunch of definitions like this coupled with a few diagrams / pictures could clarify a lot.

**Comment by ResonanceXX** [comment]
Transparency, Equity and Fairness.

And well crystallized objectives. An even more assymmetric process for problem solving and attaining our common objectives.

**Comment by ResonanceXX** [comment]
An organogram based representation of these information would provide better access to the information. Since there are several operating arms of RCHAIN, are there officials heading those?
Or are they decentralized in operations and control.

**Comment by pythonical** [comment]
I think groups like 'the Linux Foundation' and such would be better for a reference than Apple or Coke?

**Comment by dckc** [comment]
@pmoorman some of the energy around a membership working group went into #385. It's not quite what I expected, though.

Meanwhile
  - your list sure looks like the pattern language I've been working on in the wiki
  - some of these things are not like the others: Mercury isn't a governance process or a structural unit. They still fit in a pattern language, but not in an org chart.

**Comment by pmoorman** [comment]
@dckc 

> your list sure looks like the pattern language I've been working on in the wiki

Wait... **can you link me to where I can find that?**
I've seen the stuff like https://github.com/rchain/bounties/wiki/Coop-Committees, etc. 

If we can work together towards a simple, unified list... that would be great. Maybe the wiki could have an 'index page' with basic breakdown (like I wrote here), and more detailed pages for people that want to have more depth (like the committees page, for instance).

---

> some of these things are not like the others: Mercury isn't a governance process or a structural unit. They still fit in a pattern language, but not in an org chart.

Yeah, good point. **Maybe we should start with the pattern language** (cool name, I didn't know what to call it...), and then create the visuals around it once that makes sense.  Agreed?

Depending on the lense you look through, the visual would change (e.g. org chart vs. brands/subbrands, etc.)

**Comment by pmoorman** [comment]
@pythonical two very good points you make there. I was thinking Apple / Coca Cola in a discussion with @AyAyRon-P to try figure out the correct analogies, but changing to something else might make sense.

One downside with going with Linux foundation, is that for non-developers (like myself) the analogy might be harder to grasp. I just don't really know how Linux is structured.... so the analogy wouldn't help me much.  I'm open to discussing other analogies though. Suggestions welcome!

Good point about the shareholders, etc. We should change that to avoid confusion.

**Comment by AyAyRon-P** [comment]
Maybe Coca-Cola doesn't work as well but we need to keep brand comparison simple and obvious like an ELI5 to give a clear understanding of RCHAIN's - what, why, how. Once we've established that with the reader then we can begin to explain things in a more technical voice. 

@pmoorman Looking at the diagram at the top - RCHAIN COOP is the Corporation with RCHAIN as the Masterbrand below it. 

@dckc so is Mercury a product or another language like RhoLang - I like the term pattern language, that term, by itself sounds like a product series. 

Instead of shareholders, @pmoorman what about:
participant
sharer
member
co-partner(s)
representative 
affiliate

**Comment by pmoorman** [comment]
Mercury is a release (version of the software). Mercury would be followed by Venus, etc.

So the analogy of Mercury to OS X Yosemite or whatever could work (but maybe also hard to grasp for non-Apple users). 
It's definitely _not_ on the same spectrum as Rholang, because Rholang is a "thing in itself", so to speak. 

You said

> 
...RCHAIN COOP is the Corporation with RCHAIN as the Masterbrand below it.  

The problem with that is, as @pythonical points out, that RChain is of course exactly _not_ a corporation (but a coop!). So calling it a sort of corporation is confusing. That's where the Linux suggestion comes from.

**Comment by dckc** [comment]
@pmoorman I wonder in what contexts this picture will be used. Creating it is not really the end goal, is it?

Is it a 5 minute slide presentation that one member presents to another during on-boarding?

Is it a web page with lots of google-juice so that people find it when searching for stuff in the web?

**Comment by pmoorman** [comment]
@dckc agree. 

As far as I'm concerned, it could be all of the above. 
- A slide deck for a live onboarding call is a great idea. 
- A short pre-recorded webinar — that new members can watch at their leisure — is also a good idea
- (A _live_ webinar once a week or so, is probably _also_ a good idea, btw).

Definitely we need to create educational material to help in the onboarding process. But even relatively well-onboarded people like myself would benefit from additional clarity.

Right now, the blocking problem is that almost everyone is just unclear about what the structures are. That's blocking from any further action.  

**So the picture(s) and pattern language will force us into clarity.**

**Comment by dckc** [comment]
A lot of people liked the way [Peter drew a diagram](https://youtu.be/ZpJOxa15Pgc?t=48m56s) (video).

**Comment by dckc** [comment]
I ran across this while auditing for issues over 5 days old with no clear approval / rejection.

@pmoorman I see you removed **Marketing**, so you evidently don't think it's important for that goal. And yet you didn't close it as **wontfix**. Do you plan to continue working on this? If so, what's the next step?

@barneycinnamon and @jimscarver is it important for **Governance**?

@makys I don't know why I took the **greeter** label off back on Feb 27; I just put it back on. Do new members already have a clear picture of governance?

Now that I think about it, @kitblake and company made some progress on https://coop-member-v1.rchain.me/about/governance/ . I guess I'll add **member-site**. I wonder whether that draft addresses this issue, or whether we should keep this open until the member site launches? In my day job, we close individual issues optimistically when we don't see any work to do on them, and then we have a separate "validate and release" issue.

**Comment by pmoorman** [comment]
Yes, I think it's governance more than marketing (since it's not outward-facing) so I removed that label. I would say that **greeter** label is justified, because that's really the biggest use case.

I would like that people could understand at a glance how the coop relates to it's partner companies like Pithia, Reflective & Pyrofex, and also how governance works inside the coop in as much detail as we can reasonably provide.

My thinking is that the better the transparency about how governance works, the easier it will be for everyone to get things done.

I'll slap on the **needs-SMART-objectives** label for myself, because I need to think about what the best next steps are, and whether I should address them here or rather in separate actions/issues.

**Comment by barneycinnamon** [comment]
@dckc @pmoorman Definitely important for Governance in my view, but I would narrow it down a little. Among the action items, I think "draft organizational diagram" and "outline relationships between parties" have both been done at various times and certainly should be covered clearly in the member website. I recall Lawrence recently making a comment about needing to spend time explaining the relationship between Pithia and the RChain Coop, because he found people to be confused about it. I think that is worthwhile stuff to tackle, and definitely actionable, but also probably redundant to some attempts made in various places already. 

I think a separate issue is "Define the political processes," which is clearly Governance, probably interacts with Voting, and is not at all narrow or straightforward and is something that, in my view, would be good to have assigned from the Executive Committee to itself with the option to sub-contract to / bring in a couple experts (e.g. Nathan Schneider or some folks @ian-bloom has met lately on the Coop circuit) to consult or provide recommendations.
